app/util/classed.ts

```diff
@@ -6,22 +6,16 @@
  * Copyright Oxide Computer Company
  */
 import cn from 'classnames'
-import React, { forwardRef, type JSX } from 'react'
+import React, { type JSX } from 'react'
 
 // all the cuteness of tw.span`text-green-500 uppercase` with zero magic
 
 const make =
   <T extends keyof JSX.IntrinsicElements>(tag: T) =>
   // only one argument here means string interpolations are not allowed
   (strings: TemplateStringsArray) => {
-    const Comp = forwardRef(
-      ({ className, children, ...rest }: JSX.IntrinsicElements[T], ref) =>
-        React.createElement(
-          tag,
-          { className: cn(strings[0], className), ...rest, ref },
-          children
-        )
-    )
+    const Comp = ({ className, ...rest }: JSX.IntrinsicElements[T]) =>
+      React.createElement(tag, { className: cn(strings[0], className), ...rest })
```

Comment:
This one is my favorite. I changed the `children` thing from a positional arg to a prop and as far as I can tell that's totally fine. I [confirmed](https://github.com/facebook/react/blob/56c7d1070aabe42b955ea0006477ea1ca02dd0c5/packages/react/src/jsx/ReactJSXElement.js#L641-L801) it in the React source.